charts/studio/values.yaml

```diff
@@ -287,6 +287,8 @@ global:
     maxViews: "100"
     maxTeams: "10"
 
+  internalCA: ""
+#  selfSigneCAConfigMap:
```

Comment:
We might want to make the naming easier to understand for users.
My proposal:
- `internalCA` -> `customCACert`
- `selfSigneCAConfigMap` -> `customCACertConfigMap`